### PR TITLE
Adding support for toHaveUrlContaining and toHaveTitleContaining

### PR DESCRIPTION
**Pull request by mathew-jithin**

### CODE REVIEW

**Comment by mathew-jithin** [comment]
@christian-bromann @mgrybyk I was a bit not understanding the unit tests (bit new to this), so not able to write any specific tests if needed. Checked only with the existing tests and they were working fine. Please let me know if any of my considerations were wrong.

If specific are to be added, will do the same if you can help me a bit here :-)

**Comment by mathew-jithin** [comment]
Forgot to add the corresponding docs for this. Added it now and made a few changes in the existing one.

File: types/jest-global.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/// <reference types="expect-webdriverio/types/standalone"/>
```
Comment (mgrybyk) [inline types/jest-global.d.ts:1]: Please revert the change, this would break types

File: types/standalone.d.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-/// <reference types="expect-webdriverio/types/jest-expect-clone/jest-expect"/>
```
Comment (mgrybyk) [inline types/standalone.d.ts:1]: Please revert the change, this would break types

File: types/standalone-global.d.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/// <reference types="expect-webdriverio/types/standalone"/>
```
Comment (mgrybyk) [inline types/standalone-global.d.ts:1]: Please revert the change, this would break types

**Comment by mgrybyk** [comment]
Types we export should always starts with the package name

File: types/standalone-global.d.ts (hunk 3 above)
Comment (mathew-jithin) [reply types/standalone-global.d.ts]: Done!

File: types/jest-global.d.ts (hunk 1 above)
Comment (mathew-jithin) [reply types/jest-global.d.ts]: Done!

**Comment by mathew-jithin** [comment]
@mgrybyk @christian-bromann Made the changes as mentioned.

> Types we export should always starts with the package name, other than that LGTM 👍
